evap/evaluation/models.py

```diff
@@ -1645,7 +1646,70 @@ def create_superuser(self, *, email, password=None, first_name=None, last_name=N
         return user
 
 
-class UserProfile(AbstractBaseUser, PermissionsMixin):
+class EvapBaseUser(models.Model):
+    """This is strongly related to the django.contrib.auth.base_user.AbstractBaseUser model, but does not define natural_key."""
+
+    password = models.CharField(_("password"), max_length=128)
+    last_login = models.DateTimeField(_("last login"), blank=True, null=True)
+
+    # Stores the raw password if set_password() is called so that it can
+    # be passed to password_changed() after the model is saved.
+    _password = None
+
+    class Meta:
+        abstract = True
+
+    def get_username(self):
+        """Return the username for this User."""
+        return getattr(self, self.USERNAME_FIELD)
+
+    @property
+    def is_anonymous(self):
+        """
+        Always return False. This is a way of comparing User objects to
+        anonymous users.
+        """
```

Comment:
I don't think our code uses this -- why do we need it?

(I wouldn't copy the docstring from the django class)

Reply:
needed by `django.contrib.auth.checks`